string LAB_color_info
float64 lMean
float64 lStd
float64 aMean
float64 aStd
float64 bMean
float64 bStd